javascript/chronometer.js

```diff
@@ -1,26 +1,43 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
-  }
-  startClick(callback) {
-    // ... your code goes here
+    this.currentTime = 0,
+    this.miliseconds = 0,
+    this.intervalId = 0,
+    this.milisecondsIntervalId = 0
+  }
+  startClick(callbackSecs, callbackMiliSecs) {
+    this.intervalId = setInterval(()=>{
+      this.currentTime = this.currentTime +1;
+      callbackSecs();
+    },1000);
+    this.milisecondsIntervalId = setInterval(()=>{
+      this.miliseconds = this.miliseconds +1;
+      callbackMiliSecs();
+    },10)
   }
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime/60);
   }
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime - this.getMinutes()*60) 
+  }
+  getMiliSecons(){
+    if(this.miliseconds >= 100){this.miliseconds = 0}
```

Comment:
Little tip, when doing if conditionals in one line you can remove the curly brackets. Looks better and still easy to read.